tests/contrib/operators/test_dataproc_operator.py

```diff
@@ -256,13 +258,83 @@ def test_cluster_name_log_sub(self):
                 dag=self.dag
             )
             with patch.object(dataproc_task.log, 'info') as mock_info:
-                context = { 'ts_nodash' : 'testnodash'}
+                context = {'ts_nodash': 'testnodash'}
 
-                rendered = dataproc_task.render_template('cluster_name', getattr(dataproc_task,'cluster_name'), context)
+                rendered = dataproc_task.render_template(
+                    'cluster_name',
+                    getattr(dataproc_task, 'cluster_name'), context)
                 setattr(dataproc_task, 'cluster_name', rendered)
                 with self.assertRaises(TypeError) as _:
                     dataproc_task.execute(None)
-                mock_info.assert_called_with('Creating cluster: %s', u'smoke-cluster-testnodash')
+                mock_info.assert_called_with('Creating cluster: %s',
+                                             u'smoke-cluster-testnodash')
+
+
+class DataprocClusterScaleOperatorTest(unittest.TestCase):
+    # Unit test for the DataprocClusterDeleteOperator
```

Comment:
Nit. Can you change `# Unit test for the DataprocClusterDeleteOperator` to `# Unit test for the DataprocClusterScaleOperatorTest `